arch/xtensa/src/lx6/Toolchain.defs

```diff
@@ -55,8 +59,14 @@ else
   MAXOPTIMIZATION += -fomit-frame-pointer
 endif
 
-ifeq ($(CONFIG_MM_KASAN),y)
-  ARCHCPUFLAGS += -fsanitize=kernel-address
+CXXFLAGS =
+
+ifneq ($(CONFIG_CXX_EXCEPTION),y)
+  CXXFLAGS += -fno-exceptions -fcheck-new
```

Comment:
Why not to reuse `ARCHCXXFLAGS` and leave `CXXFLAGS :=` as is? I mean `ARCHCXXFLAGS += -fno-exceptions -fcheck-new`? The same as it is done few lines above for `ARCHCPUFLAGS += -fsanitize=kernel-address`

Reply:
It is also an option. At first I preferred to leave `ARCHCFLAGS` and `ARCHCXXFLAGS` restricted to the `board` definition.
But after your comment, I believe it makes sense for the `Toolchain.defs` file to define arch-common C and C++ flags, and each board will append application-specific flags.

I'll submit another patch with this approach.
Thanks for the feedback.

Reply:
Done, PTAL again.